rails/app/views/admin/stories/import_page.html.erb

```diff
@@ -0,0 +1,23 @@
+<section class="main-content__body padding-fix color-fix">
+  <p>
+    IMPORTANT: Need to import stories? The order of the values in your csv must be in a particular order. Feel free to use this <%= link_to "SAMPLE CSV", export_sample_csv_admin_stories_path %> to get started with successfully uploading stories.
+  </p>
+
+  <%= form_tag send("import_csv_admin_stories_path"), multipart: true do %>
```

Comment:
Something feels weird about this change from `import_csv_admin_#{page.resource_name.pluralize}_path` to `import_csv_admin_stories_path`

Reply:
@Aliciawyse i think thats totally fine! We're moving from a generic form to forms that are more specific to the resource's need. I've found the administrate abstraction to usually get in the way anyway; anything thats more specific and rails-y feels a little easier to read too. 

If we do end up wanting to generecize it again, we could always find the common pieces between the forms and make a partial that goes back to the `import_csv_admin_#{page.resource_name.pluralize}_path` style.

Reply:
@Aliciawyse I agree with @FeminismIsAwesome in this; the form tag itself is going to be "duplicated" across each specialized view. The Administrate "way" forced us to do weird things like interpolating the resource into the path and I for one am glad to see that go.

IF we wanted to continue to share some of the logic, we could create a partial view for the form; but I'm okay with waiting on this until we see the repetition.